### PR TITLE
Add textFields border style

### PR DESCRIPTION
**Pull request by ehab-al-cko**
[Jira Ticket ](https://checkout.atlassian.net/browse/PIMOB-1240)

## Proposed changes

For textFields Change the border edges: 
cornerRadius and borderWidth properties should be injected and allowed by merchants to be customised.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [x] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [x] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by alex-ioja-yang-cko — APPROVED** [review]
Looks good, thank you for further improving the code clarity by finding some logic to better group the properties in the objects previously created 👍

File: Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift [hunk 1]
```diff
@@ -29,12 +29,6 @@ class BillingFormButtonViewTests: XCTestCase {
         XCTAssertEqual(view.image?.tintColor, style.button.disabledTintColor)
     }
 
-    func testButtonStyle() {
-        XCTAssertEqual(view.button?.isEnabled, style.button.isEnabled)
-        XCTAssertEqual(view.button?.tintColor, .clear)
-        XCTAssertEqual(view.button?.backgroundColor, .clear)
-    }
```
[the remark this answers is not in the document]
Comment (ehab-al-cko) [reply Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift:36]: it was duplicate code

**Comment by deepesh-vasthimal-cko** [comment]
@ehab-al-cko if you look into my branch origin/feature/PIMOB-1237_Custom1_Custom2_SampleApp. commit `05939fec7ad55e387ddc83238f5a4b229c198487` I had to set few setters for ElementErrorViewStyle and ElementStyle for the sample app to configure properties. see below. I think you may have to change this somewhere. 

Screenshot 2022-06-09 at 13 36 43
Screenshot 2022-06-09 at 13 36 47

**Review by deepesh-vasthimal-cko — APPROVED** [review]
Just one comment from my sample app branch about setters.

**Review by harry-brown-cko — APPROVED** [review]
I've added a comment on the API spec about using `placeholder` instead of `placeHolder`. We should change this if the team agrees

https://checkout.atlassian.net/wiki/spaces/FS/pages/5266440242?focusedCommentId=5290689012#comment-5290689012

File: Source/UI/NewUI/BillingForm/Default Implementation/Elements/TextField/DefaultTextField.swift [hunk 2]
```diff
@@ -2,18 +2,21 @@ import UIKit
...
-    public var isPlaceHolderHidden: Bool  = false
-    public var font: UIFont = UIFont(graphikStyle: .regular, size: Constants.Style.BillingForm.InputTextField.fontSize.rawValue)
     public var textColor: UIColor = .codGray
     public var normalBorderColor: UIColor = .mediumGray
     public var focusBorderColor: UIColor = .brandeisBlue
     public var errorBorderColor: UIColor = .tallPoppyRed
     public var backgroundColor: UIColor = .white
     public var tintColor: UIColor = .codGray
+    public var borderWidth: CGFloat = 1.0
+    public var cornerRadius: CGFloat = 10.0
```
[the remark this answers is not in the document]
Comment (ehab-al-cko) [reply Source/UI/NewUI/BillingForm/Default Implementation/Elements/TextField/DefaultTextField.swift:17]: I know. For Float numbers, i prefer to make it x.x

**Pull request merged**